### ISSUE TITLE
TypeError: cheerio is not a function

### ISSUE THREAD

**Issue by trevorr** [anvilco/spectaql#44]
I'm trying out spectaql (0.2.0) for the first time, and I'm hitting the following error:

```
$ npx spectaql docs/spectaql.yml

COULD NOT LOAD EXAMPLE PROCESSOR

Running "sass:full" (sass) task

Running "sass:foundation" (sass) task

Running "concat:css" (concat) task

Running "cssmin:css" (cssmin) task
>> 1 file created. 140.88 kB → 121.62 kB

Running "concat:js" (concat) task

Running "uglify:js" (uglify) task
>> 1 file created 6.82 kB → 3 kB

Running "clean:html" (clean) task
>> 0 paths cleaned.

Running "compile-handlebars:compile" (compile-handlebars) task
Task error: [TypeError: cheerio is not a function
  at Object.markdown (/Users/trobinson/fittfinder/fittfinder-service/node_modules/spectaql/app/lib/common.js:84:15)
```

I tried running `npm i -D cheerio`, but got the same error. I'm running against a schema file:

```yaml
introspection:
  schemaFile: ./dist/schema.graphql

servers:
  - url: https://example.com/graphql
    description: Production
    production: true
```

I tried with a trivial schema too, and got the same error:

```graphql
schema {
  query: Query
}

type Query {
  test: Int
}
```

node/npm/OS version:

```
$ node -v
v12.18.4
$ npm -v
6.14.6
$ uname -v
Darwin Kernel Version 20.3.0: Thu Jan 21 00:07:06 PST 2021; root:xnu-7195.81.3~1/RELEASE_X86_64
```

**Comment by benhickson** [comment]
Hi Anvil, just opened a PR which should fix the issue that @trevorr and I are experiencing.

@trevorr , I got this working locally by going into `node_modules/spectaql/app/lib/common.js` and appending `.default` to line 3:
```
const cheerio = require('cheerio').default
```
Not sure the best way to fix it for real, but possibly if my PR gets merged that might do it.

**Comment by newhouse** [comment]
After digging in to the root cause, I came across this: https://github.com/cheeriojs/cheerio/releases

Which shows that cheerio released `1.0.0-rc.9` a few days ago. So, I'm guessing that the problem for some began because the `package.json` specifies `"cheerio": "^1.0.0-rc.2"`...but if you do a new install today, you will actually get `"1.0.0-rc.9"`.

In release `v1.0.0-rc.9`, they list the following breaking change:
> Breaking change: If you were using the function exported by Cheerio directly instead of first load()ing a document, you will now have to update the require to use the default export.

Which is right onto what @benhickson was getting after.

However, it then goes on to state:
> Please note that this way of using Cheerio is deprecated and might be removed in a future version. Please consider updating your code to:
> ```js
> const cheerio = require("cheerio");
> const $ = cheerio.load(dom)
> $('div')
> ```

So, I've opted to change the code to the way suggested in the docs vs just adding the `.default` to the `require`. Feels like the better way to go since I saw that.

Thanks for bringing this up!

**Comment by benhickson** [comment]
Looks good! thanks @newhouse

**Comment by trevorr** [comment]
Thanks @newhouse, can't wait to try it out! Also, you're a towel. 😂

**Comment by benhickson** [comment]
@newhouse When should we expect this to arrive on NPM?

**Comment by newhouse** [comment]
> @newhouse When should we expect this to arrive on NPM?

Fix is in `0.2.1` available now. @benhickson
